README.md

```diff
@@ -51,6 +52,14 @@ $ ./tools/sof-dump-status.py -p
 apl
 ```
 
+### test case result
+| exit code | display | discription            |
+| --------- | ------- | ---------------------- |
+| 0         | PASS    | Test has passed        |
+| 1         | FAIL    | Test has failed        |
+| 2         | N/A     | Test is not applicable |
+| *         |         | unknow exit status     |
```

Comment:
Where is the "incomplete" status ? i.e. the CI has failed or had an error.
I would also add a link to this README (in git) from the deviec test reulst page too (so that any user can uderstand the results).

Reply:
@lgirdwood , sof-test is not CI. We separate them very clearly. Here is only the "sof-test" status.  At the end, you will see more status on CI result page in the CI code.

Reply:
I think this still needs the "incomplete" status e.g. what happens if the script cant access a resource needed to perform the test ? This way the result is correctly propagated up the stack through the CI to the developer.

Reply:
@lgirdwood if the resource on that DUT is not access for this case, I believe it will become N/A